manifests/dashboard-install.yaml

```diff
@@ -57,6 +57,8 @@ rules:
   verbs:
   - get
   - update
+  - list
+  - watch
```

Comment:
These changes you have to provide for `dashboard-install/dashboard-clusterrole.yaml` file. That's the source of the changes. 
The dashboard-install.yaml is generated from the `dashboard-install` folder.